bench/run_benchmarks.py

```diff
@@ -66,13 +66,7 @@ def create_channel(self):
                     'type': 'dmark',
                     'config': {
                         'web_path': 'api',
-                        'web_port': 8001,
-                        "redis_manager": {
-                          "key_prefix": "my_ussd",
-                          "host": "redis",
-                          "db": 2,
-                          "port": 6379
-                        }
```

Comment:
Why are we getting rid of this redis configuration?

Reply:
I added it when I was running redis, rabbit and Junebug using docker-compose. I ended up running `jb` directly which meant the default `localhost` was fine.